tutorial/tutorial-template-guide.md

```diff
@@ -0,0 +1,132 @@
+# The tutorial template
+
+The tutorial template includes:
+
+* Placeholder text that you can replace with your own text. This text is indicated in {curly braces}.
+* Writing instructions, also in {curly braces}.
```

Comment:
**Praise:** I really like this introduction you've added here for clarity. We should consider making this boilerplate text part of all the about guides.

**Chore:** I notice there's some discrepancies between formatting in your template vs. the quickstart template. The template working group leads might need to meet and agree on our conventions, then add them to a style guide.